kafka/pykafka/balancedconsumer.py

```diff
@@ -0,0 +1,371 @@
+import itertools
```

Comment:
I believe it can, but doesn't that make the c implementation more of a hybrid of python and c? The BalancedConsumer is python and the SimpleConsumer is c.